### ISSUE TITLE
With GCC, -fPIC should not be specified on static builds

### ISSUE THREAD

**Issue by lvella** [ValveSoftware/openvr#1458]
I saw you don't take pull requests, so I won't bother, but it would be nice if `-fPIC` was not specified during static builds, as it complicates the assembly needlessly.

**Comment by TobiSchluter** [comment]
Don't do that please or at least make it optional because it would prevent inclusion of the static library into DLLs. It's also not like there is any performance-critical code in the library where PIC would be detrimental.

**Comment by lvella** [comment]
Inside .so, you mean? Because everywhere I looked says DLLs do not need PIC.

Also, it is very easie to add to create a CMake configuration adding `-fPIC` to CXXFLAGS, but it is impossible to remove if it is hardcoded in the CMakeLists.txt.

**Comment by TobiSchluter** [comment]
Yes, sorry, I was using "DLL" generically which was ill-advised in this context. I certainly agree that the CMakeLists don't follow best practices :)
